File: test_commits/java/feature_addition_original4.java
import java.util.ArrayList;
import java.util.List;

class Task {
    private String name;
    private boolean completed;

    public Task(String name) {
        this.name = name;
        this.completed = false;
    }

    public String getName() {
        return name;
    }

    public boolean isCompleted() {
        return completed;
    }
}

class TaskManager {
    private List<Task> tasks = new ArrayList<>();

    public void addTask(String name) {
        tasks.add(new Task(name));
    }

    public List<Task> getAllTasks() {
        return tasks;
    }
}

File: test_commits/python/feature_addition_original4.py
class Task:
    def __init__(self, name):
        self.name = name
        self.completed = False

class TaskManager:
    def __init__(self):
        self.tasks = []

    def add_task(self, name):
        task = Task(name)
        self.tasks.append(task)

    def get_all_tasks(self):
        return self.tasks
